tests/test_raster/test_raster.py

```diff
@@ -971,7 +964,6 @@ def test_copy(self, example: str) -> None:
 
         # Open data, modify, and copy
         r = gu.Raster(example)
-        r.data += 5
```

Comment:
The test does not fit the comment anymore, as the raster is not modified before being copied...

Reply:
Added back! Good catch.